cluster/aws/util.sh

```diff
@@ -1491,7 +1491,7 @@ function test-build-release {
 # Assumed vars:
 #   Variables from config.sh
 function test-setup {
-  "${KUBE_ROOT}/cluster/kube-up.sh"
+  . "${KUBE_ROOT}/cluster/kube-up.sh"
```

Comment:
Without it, subsequent calls to things like `detect-security-group` will fail as `VPC_ID` is set within `kube-up` and hence will not be availabile if `kube-up.sh` executes in a sub-process.